docs/how-to/how-to-write-fuzz-tests.md

```diff
@@ -0,0 +1,91 @@
+This document introduce how to write fuzz tests in GreptimeDB.
+
+All fuzz test-related resources are located in the `/tests-fuzz` directory.
+- Each target(located in the `/tests-fuzz/targets`) stands for an independent fuzz test case.
+- Fuzz test fundamental components are located in the `/tests-fuzz/src`.
+
+Figure 1 illustrates that it utilizes a Rng (Random Number Generator) to generate the IR (Intermediate Representation), then employs a DialectTranslator to produce specified dialects for different protocols.
```

Comment:
Produce specified dialects for different protocols., then what? Looks like missing something.